docs/LOCALIZATION.md

```diff
@@ -0,0 +1,248 @@
+# Localization
+
+Starting from Web Chat 4.8, a dedicated team at Microsoft will start localizing our UI for 44 languages.
+
+## Adding a new language
+
+To add a new language, please update following files:
+
+-  Update `/packages/component/src/Localization/overrides.json` and add the language according to Unicode CLDR.
+   -  `GLOBALIZE_LANGUAGE` is the language code used for referencing Unicode CLDR
+      -  To see the supported list of Unicode CLDR, look under `/packages/component/node_modules/cldr-data/main/`.
+   -  (Optional) `TEXT_TO_SPEECH` is used to indicate the language code for speech.
+      -  Some locales has different oral but share same written language with other locales. For example, written language in Hong Kong and Taiwan are Traditional Chinese, while oral languages are Cantonese and Taiwanese Mandarin respectively.
+      -  The language code used in Web Chat only indicate the written language.
+      -  Thus, when performing text-to-speech, the language code will be remapped for oral language.
+      -  For example, written language in Hong Kong is `zh-Hant-HK`, while the oral language is `zh-HK` (Cantonese). And the written language in Taiwan is `zh-Hant`, while the oral language is `zh-TW` (Taiwanese Mandarin).
+   -  (Optional) `COGNITIVE_SERVICES_*` is used to indicate whether the language supported Cognitive Services Speech Services.
+      -  `COGNITIVE_SERVICES_SPEECH_TO_TEXT` is used to indicate the language is supported by Cognitive Services Speech-to-Text Service.
+      -  `COGNITIVE_SERVICES_TEXT_TO_SPEECH` is used to indicate the language is supported by Cognitive Services Text-to-Speech Service. If the service support neural voices, set it to `"neural"`, otherwise, `true`
```

Comment:
```suggestion
      -  `COGNITIVE_SERVICES_TEXT_TO_SPEECH` is used to indicate that the language is supported by Cognitive Services Text-to-Speech Service. If the service supports neural voices, set it to `"neural"`; otherwise, `true`
```